src/fides/api/task/manual/manual_task_utils.py

```diff
@@ -283,8 +284,40 @@ def get_manual_task_instances_for_privacy_request(
     )
 
 
+def filter_manual_task_configs_by_policy(
```

Comment:
Tiny nit: I would consider naming this something like `manual_task_configs_allowed_by_policy` - filter is generally used for filtering a set of data where you might pass in the configs as well and get back the valid configs instead of returning a bool